.github/scripts/check-pr.js

```diff
@@ -46,6 +46,12 @@ async function run() {
             throw new Error('The PR has no milestone.');
         }
 
+        // Thirdparty dependencies are not controlled by us so we will skip the issue matching check for them"
+        if (prTitle.includes("build(deps):")) {
```

Comment:
instead of checking if prTitle includes `build(deps)s`, we might want to use a different criterion, such as checking the PR author or something like that. The build(deps) title doesn't necessarily indicate that the PR is from a third party, so relying solely on that check might not be sufficient IMO

Reply:
Updated.